File: DotnetSever/Models/Helicopter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;

namespace RSIVueloAPI.Models
{
    // mimics Helicopter stucture in Helicopter.java
    public class Helicopter
    {
        public string Type { get; set; }
        public string Model { get; set; }
        public double CapacityWeight { get; set; }
        public double CrewMax { get; set; }
        public double CrewMin { get; set; }
        public double FuselageLength { get; set; }
        public double Height { get; set; }
        public double RotorDiameter { get; set; }
    }
}

File: heliJava/Helicoper2.0/Helicoper2.0/src/main/java/com/vuelo/Helicopter20/Entities/Helicopter.java
package com.vuelo.Helicopter20.Entities;
import lombok.*;
import org.bson.types.ObjectId;
import org.springframework.data.annotation.Id;
import org.springframework.data.annotation.Transient;
import org.springframework.data.mongodb.core.mapping.Document;
import org.springframework.data.mongodb.core.mapping.Field;
import org.springframework.data.mongodb.core.mapping.MongoId;

import java.util.Objects;

@Document(collection = "Helicopters")
@Setter
@Getter
@ToString
@AllArgsConstructor
@Builder
public class Helicopter {
    String _id;
    String type;
    @Field
    String model;
    String capacityWeight;
    String crewMax;
    String crewMin;
    String fuselageLength;
    String height;
    String rotorDiameter;
    String maxSpeed;
    String url;

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Helicopter that = (Helicopter) o;
        return Objects.equals(type, that.type) &&
                model.equals(that.model) &&
                Objects.equals(capacityWeight, that.capacityWeight) &&
                Objects.equals(crewMax, that.crewMax) &&
                Objects.equals(crewMin, that.crewMin) &&
                Objects.equals(fuselageLength, that.fuselageLength) &&
                Objects.equals(height, that.height) &&
                Objects.equals(rotorDiameter, that.rotorDiameter) &&
                Objects.equals(maxSpeed, that.maxSpeed);
    }

    @Override
    public int hashCode() {
        return Objects.hash(type, model, capacityWeight, crewMax, crewMin, fuselageLength, height, rotorDiameter, maxSpeed);
    }


}
